src/Command/RemindEndingCfpCommand.php

```diff
@@ -49,7 +49,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         foreach ($conferences as $conference) {
             $remainingDays = (int) ($conference->getCfpEndAt()->diff($today)->format('%a'));
 
-            if (\in_array($remainingDays, self::REMAINING_DAYS_STEPS, true)) {
+            if (\in_array($remainingDays, self::REMAINING_DAYS_STEPS, true) && !$conference->getExcluded()) {
```

Comment:
It could be better to filter directly at SQL level, we do not need to fetch from database excluded conferences.